core/tests/org.fusesource.ide.camel.model.service.core.tests/META-INF/MANIFEST.MF

```diff
@@ -7,4 +7,5 @@ Fragment-Host: org.fusesource.ide.camel.model.service.core;bundle-version="8.0.0
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Require-Bundle: org.assertj.core;bundle-version="2.1.0",
  org.junit;bundle-version="4.12.0",
- org.jboss.tools.locus.mockito;bundle-version="1.9.5"
+ org.jboss.tools.locus.mockito;bundle-version="1.9.5",
+ org.fusesource.ide.camel.model.service.v2_16_1;bundle-version="8.0.0"
```

Comment:
We should try to not use / rely on the service implementation bundles. Service-core should be all we work with.